File: TypeScript/Problem1.ts
function findMedianSortedArrays(nums1: number[], nums2: number[]): number {
    const mergedArray = [...nums1, ...nums2].sort((a, b) => a - b);
    const length = mergedArray.length;
    if (length % 2 !== 0) {
        return mergedArray[Math.floor(length / 2)];
    }
    else{
        return (mergedArray[length / 2 - 1] + mergedArray[length / 2]) / 2;
    }
};

File: Python/Problem1.py
#Median Of Two Sorted Array
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        arr = nums1 + nums2
        mid = len(arr) // 2 
        arr.sort()
        if len(arr) % 2 == 0:
            return (arr[mid] + arr[mid - 1]) / 2    
        return arr[mid]
            